readthedocs/templates/projects/project_translations_form.html

```diff
@@ -0,0 +1,48 @@
+{% extends "projects/project_edit_base.html" %}
+
+{% load i18n %}
+
+{% block title %}{% trans "Add Translation" %}{% endblock %}
+
+{% block nav-dashboard %} class="active"{% endblock %}
+
+{% block editing-option-edit-translations %}class="active"{% endblock %}
+
+{% block project-translations-active %}active{% endblock %}
+{% block project_edit_content_header %}{% trans "Add Translation" %}{% endblock %}
+
+{% block project_edit_content %}
+  <p class="help_text">
+    {% blocktrans trimmed %}
+      To add a translation to your project, select a project below.
+    {% endblocktrans %}
+  </p>
+
+  {% if not project.supports_translations %}
+    {% url 'projects_edit' project_slug=project.slug as project_settings_url %}
+    <p>
+      {% blocktrans trimmed with project_settings_url=project_settings_url %}
+      This project is <a href="{{ project_settings_url }}">configured</a> with a versioning scheme that doesn't support translations.
+      {% endblocktrans %}
+    </p>
+  {% elif project.main_language_project %}
+    <p>
+      {% blocktrans trimmed with language=project.get_language_display  main_project=project.main_language_project.name %}
+        This project is already configured as the <code>{{ language }}</code> translation of
+        "{{ main_project }}".
+        Nested translations are not supported.
+      {% endblocktrans %}
+    </p>
```

Comment:
I'm noting these few blocks here as good logic to put into the Form class instead, as pre-validation/validation errors. Just noting this for now, we don't need to alter things in this PR.